app/presenters/sufia/file_set_presenter.rb

```diff
@@ -13,6 +13,16 @@ def tweeter
       end
     end
 
+    def rights
+      return nil if solr_document.rights.nil?
```

Comment:
Can omit the explicit `nil` here, like you did in `render_visibility_badge` above.